hydromt/nodata.py

```diff
@@ -0,0 +1,22 @@
+"""Functions to handle nodata values.""" ""
+from enum import Enum
+from logging import Logger
+
+from .exceptions import NoDataException
+
+
+class NoDataStrategy(Enum):
+    """Strategy to handle nodata values."""
+
+    RAISE = "raise"
+    IGNORE = "ignore"
+
+
+def _exec_strat(msg: str, strategy: NoDataStrategy, logger: Logger) -> None:
```

Comment:
Could you rename it to `_exec_nodata_strat`? out in the wild it's not that clear what it's actually referring too.